com/course/zenkouv/src/Lecture3/Task1/KnightApp.java

```diff
@@ -5,20 +5,26 @@
 import Lecture3.Task1.util.calculator.OutfitCalculator;
 import Lecture3.Task1.util.filter.OutfitFilter;
 import Lecture3.Task1.util.helper.OutfitHelper;
+import Lecture4.task2.factory.OutfitReaderFactory;
+import Lecture4.task2.factory.WriterFactory;
+import Lecture4.task2.util.operation.FileOperation;
+import Lecture4.task2.util.writer.CustomWriter;
 
+import java.io.IOException;
 import java.util.List;
 
 class KnightApp {
 
     private static final Knight ARTUR = KnightFactory.createEquippedKnight("Artur");
+    private static final Knight LANCELOT = new Knight("Lancelot");
     private static final int MAX_COST = 100;
     private static final int MIN_COST = 30;
     private static final int MIN_DAMAGE = 100;
     private static final int MAX_DAMAGE = 900;
     private static final int MIN_DEFENCE = 15;
     private static final int MAX_DEFENCE = 50;
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
```

Comment:
please handle this exception in correct place. 
program should exit with correct message

Reply:
fix